openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/BallerinaClientGenerator.java

```diff
@@ -258,7 +260,8 @@ private ClassDefinitionNode getClassDefinitionNode() throws BallerinaOpenApiExce
         NodeList<Token> classTypeQualifiers = createNodeList(
                 createToken(ISOLATED_KEYWORD), createToken(CLIENT_KEYWORD));
         return createClassDefinitionNode(metadataNode, createToken(PUBLIC_KEYWORD), classTypeQualifiers,
-                createToken(CLASS_KEYWORD), className, createToken(OPEN_BRACE_TOKEN),
+                createToken(CLASS_KEYWORD), isPlugin ? createIdentifierToken("'client") : className,
```

Comment:
Is this client name finalised?  Shouldn't we use title case for the generated client class name (as thats the normal convention we have)?

Reply:
Yeah, this was discussed during a call and it was decided to use `client` in the jBallerina implementation.